streams/src/test/java/org/apache/kafka/streams/integration/ResetIntegrationTest.java

```diff
@@ -64,4 +70,70 @@ public void testReprocessingFromScratchAfterResetWithIntermediateUserTopic() thr
     public void testReprocessingFromScratchAfterResetWithoutIntermediateUserTopic() throws Exception {
         super.testReprocessingFromScratchAfterResetWithoutIntermediateUserTopic();
     }
+
+    @Test
+    public void testReprocessingFromLatestEarliestAfterResetWithoutIntermediateUserTopic() throws Exception {
+        super.testReprocessingFromLatestAfterResetWithoutIntermediateUserTopic();
+    }
+
+    @Test
+    public void testReprocessingFromOffsetAfterResetWithoutIntermediateUserTopic() throws Exception {
+        super.testReprocessingFromOffsetAfterResetWithoutIntermediateUserTopic();
+    }
+
+    @Test
+    public void testReprocessingByShiftPositiveAfterResetWithoutIntermediateUserTopic() throws Exception {
+        super.testReprocessingByShiftPositiveAfterResetWithoutIntermediateUserTopic();
+    }
+
+    @Test
+    public void testReprocessingByShiftNegativeAfterResetWithoutIntermediateUserTopic() throws Exception {
+        super.testReprocessingByShiftNegativeAfterResetWithoutIntermediateUserTopic();
+    }
+
+    @Test
+    public void testReprocessingFromFileAfterResetWithoutIntermediateUserTopic() throws Exception {
+        super.testReprocessingFromFileAfterResetWithoutIntermediateUserTopic();
+    }
+
+    @Test
+    public void testReprocessingFromDateTimeAfterResetWithoutIntermediateUserTopic() throws Exception {
+        super.testReprocessingFromDateTimeAfterResetWithoutIntermediateUserTopic();
+    }
+
+    @Test
+    public void testReprocessingByDurationAfterResetWithoutIntermediateUserTopic() throws Exception {
+        super.testReprocessingByDurationAfterResetWithoutIntermediateUserTopic();
+    }
+
+    @Test
+    public void testDateTimeFormats() throws ParseException {
```

Comment:
this should be multiple tests (one test method per scenario). Also, use better method names, eg, `shouldAcceptValidDateFormats`, `shouldThrowOnInvalidDateFormat` etc

Reply:
Yes, I will add them.